src/templates/app/cashier/payment_methods.jsx

```diff
@@ -496,13 +496,13 @@ const PaymentMethods = () => {
                                 row: [
                                     { text: <PaymentLogo logo='tether' /> },
                                     { attributes: { colSpan: 5, className: 'toggler' }, custom_td : <CustomTableData data={[
-                                        { td: it.L('Tether is a blockchain-based cryptocurrency whose cryptocoins in circulation are backed by an equivalent amount of traditional fiat currencies. For more info, please visit [_1].', '<a href="https://litecoin.org" target="_blank">https://tether.to</a>') },
+                                        { td: it.L('Tether is a blockchain-based cryptocurrency whose cryptocoins in circulation are backed by an equivalent amount of traditional fiat currencies. For more info, please visit [_1].', '<a href="https://tether.to" target="_blank">https://tether.to</a>') },
```

Comment:
could you refactor this to use the `createLink` function ? 
e.g. https://github.com/binary-com/binary-static/blob/66dbabf2fab2ab87653508dbdf72bae532fdee0e/src/templates/app/cashier/payment_methods.jsx#L461